packages/core/test/toast/toasterTests.ts

```diff
@@ -121,4 +121,17 @@ describe("Toaster", () => {
             "mutation side effect!",
         );
     });
+
+    describe("with autoFocus set to true", () => {
```

Comment:
we need to set the toaster with `autoFocus: true`, I could turn it on for all the above too, but it's not the default so a new block makes sense